products/experiments/backend/max_tools.py

```diff
@@ -1,20 +1,154 @@
-"""
-MaxTool for AI-powered experiment summary.
-"""
-
-from typing import Any
+from typing import Any, Literal
 
+from posthoganalytics import capture_exception
 from pydantic import BaseModel, Field
 
 from posthog.schema import MaxExperimentSummaryContext
 
-from posthog.exceptions_capture import capture_exception
+from posthog.models import Experiment, FeatureFlag
+from posthog.sync import database_sync_to_async
 
 from ee.hogai.llm import MaxChatOpenAI
 from ee.hogai.tool import MaxTool
 
 from .prompts import EXPERIMENT_SUMMARY_BAYESIAN_PROMPT, EXPERIMENT_SUMMARY_FREQUENTIST_PROMPT
 
+
+class CreateExperimentArgs(BaseModel):
+    name: str = Field(description="Experiment name - should clearly describe what is being tested")
+    feature_flag_key: str = Field(
+        description="Feature flag key (letters, numbers, hyphens, underscores only). Will create a new flag if it doesn't exist."
```

Comment:
"Will create a new flag if it doesn't exist." I think this is false, before I pushed the change to mount `create_featurer_flags` here, this tool failed due to flag missing.